tests/server/cluster/cluster_work_test.go

```diff
@@ -100,9 +101,6 @@ func TestAskSplit(t *testing.T) {
 		Region: regions[0].GetMeta(),
 	}
 
-	_, err = rc.HandleAskSplit(req)
```

Comment:
Why remove it?

Reply:
moved to below lines

Reply:
see https://github.com/tikv/pd/pull/5485/files/f51c150272c973e1a80f3f297866f8d1082ee6a6#diff-cc382401991df4c6e346c3c68cc95ee7754fed058699bd47a000db7fd005ea1dR121-R122